editors/mindforger/Portfile

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github  1.0
+PortGroup               qt5     1.0
+
+github.setup            dvorka mindforger 1.52.0
+revision                0
+
+homepage                https://www.mindforger.com
+
+description             Thinking notebook and Markdown editor
+
+long_description        MindForger is an open, free, well-performing Markdown \
+                        editor which respects your privacy and enables \
+                        security.  It is actually more than an editor or IDE \
+                        - it's a human mind -inspired personal knowledge \
+                        management tool.
+
+categories              editors office
+license                 GPL-2
+platforms               darwin macosx
+
+maintainers             {gmail.com:herby.gillot @herbygillot} \
+                        openmaintainer
+
+fetch.type              git
+
+depends_build-append    port:ccache \
+                        port:cmake
+
+depends_lib-append      port:zlib
+
+qt5.depends_component   qtwebengine
+
+set _cc_tmp_path        ${workpath}/.home/.ccache
+set _cm_gfm_path        ${worksrcpath}/deps/cmark-gfm/build
+set _cmake_bin          ${prefix}/bin/cmake
+
+configure.cmd           ${qt_qmake_cmd}
+configure.pre_args
+configure.args          -r ./mindforger.pro
+configure.env           CCACHE_DIR=${_cc_tmp_path}
+
+build.env-append        CCACHE_DIR=${_cc_tmp_path}
+
+post-extract {
+    system -W ${worksrcpath} "git submodule update --init --recursive"
```

Comment:
this is possible, but will not be overly popular for all the usual reasons. No option?